### DISCUSSION TITLE
thảo luận làm đê

### DISCUSSION THREAD

**Discussion by 10A2DucAn01** [phhnhan-10A02-stt21/lop-10A02-Nhom8#23] [General]
lên idea thoi ạ^^

**Comment by 10A2DucAn01** [comment]
tớ định phân tích bài thơ xong rút ra mấy thực trạng hiện giờ r làm content về mấy thực trạng đó mấy cậu thấy xiao

**Comment by NKLinhNgan-10A02-stt19** [comment]
cũng đựt, kêu amai viết đy r t vs hnhan làm phần trình bày word. Ai k làm thì đóng t 1 củ r t gánh cho::::))))))))

**Reply by 10A2DucAn01** [reply]
khôn qá
ô kê để t dới amai viết ròi gửi file word qua

**Comment by 10A2DucAn01** [comment]
t thấy chủ đề trong bài thơ là áp lực học tập nè, ai recommend câu chiện nào nói về vượt qua áp lực và nỗ lực tiến tới thành công đi^^

**Reply by NKLinhNgan-10A02-stt19** [reply]
lấy t nè::)))

**Reply by NKLinhNgan-10A02-stt19** [reply]
ô kê gửi bản word lên đây đi đan

**Reply by 10A2DucAn01** [reply]
gửi r ạ

**Reply by 10A2DucAn01** [reply]
để lấy câu chiện bạn lngan sì trét tổ hợp tự nhiên :)))

**Reply by NKLinhNgan-10A02-stt19** [reply]
và bạn lngan vượt qua bằng cách qua đời::))

**Reply by DoMinhQuanC2** [reply]
chào các bợn và sau 7749 lần ko hiểu tại sao mình ko thấy nhóm thì cuối cùng mình cũng đã vào đc ròi này

**Reply by NKLinhNgan-10A02-stt19** [reply]
cảm động qa ạ

**Reply by NKLinhNgan-10A02-stt19** [reply]
đọc bài word đi ạ

**Reply by 10A2DucAn01** [reply]
đọc word ròi cho càm nhận của cả 3 khổ đi, t note thêm vô lun

**Comment by LBUPhuong** [comment]
ủa là mỗi đoạn lồng ghép 1 câu chuyện đko

**Reply by 10A2DucAn01** [reply]
đúng ròi vậy cũng đc
